.github/workflows/ci-test.yml

```diff
@@ -19,10 +19,10 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
-      - name: Set up Node.js 18
+      - name: Set up Node.js 19
```

Comment:
Do we really want to use dev Node version that is already dead (no updates/no security updates)?

Reply:
Oops, it's already dead